src/particles/ImpactXParticleContainer.H

```diff
@@ -94,6 +93,8 @@ namespace impactx
          * @param px momentum in x
          * @param py momentum in y
          * @param pz momentum in z
+         * @param qm charge over mass in 1/eV
+         * @param bchchg total charge within a bunch in C
```

Comment:
Isn't the unit in `q_e`? If it's C then I think we need to add a scaling factor in the weighting below?
```suggestion
         * @param bchchg total charge within a bunch in q_e
```